hpat/tests/test_series.py

```diff
@@ -285,6 +285,39 @@ def test_impl(A):
         S = pd.Series(['aa', 'bb', 'cc'])
         np.testing.assert_array_equal(hpat_func(S), test_impl(S))
 
+    def test_series_copy_int1(self):
+        def test_impl(A):
+            return A.copy()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([1, 2, 3])
+        np.testing.assert_array_equal(hpat_func(S), test_impl(S))
+
+    def test_series_copy_deep(self):
+        def test_impl(A, deep):
+            return A.copy(deep=deep)
+        hpat_func = hpat.jit(test_impl)
+
+        for S in [
+            pd.Series([1, 2]),
+            pd.Series([1, 2], index=["a", "b"]),
+        ]:
+            with self.subTest(S=S):
```

Comment:
same comment as in your previous PR

Reply:
Sorry, what is wrong with this code? Previous PR was about `... if deep else ...`. This is not the same.

Reply:
The output of Series because of subTest() is ok for all output lines except the first one. The first one is not with others:
```
======================================================================
FAIL: test_series_copy_deep (hpat.tests.test_series.TestSeries) (deep=False, S=a    1
b    2
dtype: int64)
----------------------------------------------------------------------
```
I think it is pretty readable and is not printed at all if tests pass.